servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcExecutionStrategies.java

```diff
@@ -133,7 +135,9 @@ public Builder offloadNone() {
          *
          * @param executor {@link Executor} to use.
          * @return {@code this}.
+         * @deprecated Set the executor to use on the {@link io.servicetalk.transport.api.ExecutionContext}.
```

Comment:
`ExecutionContextBuilder` is an internal API. How users can set an executor for the client or server?

Reply:
The will set it on the `HttpServerBuilder `or `*ClientBuilder`. Perhaps add the missing Executor setters in this same change? I had intended to keep this PR only deprecations but they can be added.

Reply:
It's hard to deprecate without providing a working alternative for users to migrate to. If this is too complicated to provide executor setters on client/builder without the whole change of #1695, we can skip back-porting it into 0.41 branch. 0.42 will anyway have some deprecated API for serialization.

Reply:
I mean not just a setter, but a setter that will work as expected.

Reply:
We can keep deprecation of methods that are going away without replacement.

Reply:
Added `executor(Executor executor)` to builders

Reply:
The setter will work as expected. The existing implementation will use the Executor from the strategy in preference to the Executor set in the builder. In the #1695 implementation only the builder Executor will be used.